src/Mocks/VisualStudio/project.15.0.json

```diff
@@ -1,21 +1,21 @@
 ﻿{
   "dependencies": {
     "MicroBuild.Core": "0.2.0",
-    "Microsoft.VisualStudio.CoreUtility": "15.0.25824-RC",
+    "Microsoft.VisualStudio.ComponentModelHost": "15.0.25824-RC",
```

Comment:
Condition above includes `Microsoft.VisualStudio.ComponentModelHost` only for VS14. Do w still need it in `project.json`?